### ISSUE TITLE
Error when using connectionString in forRoot instead of .env

### ISSUE THREAD

**Issue by jogelin** [nestjs/azure-database#51]
## I'm submitting a...

<pre><code>
[ ] Regression 
[X] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
Based on your doc, it should be possible to use a forRoot on the main module to configure the connection but when I use it:

**app.module.ts:**
```
@Module({
  imports: [
    AzureTableStorageModule.forRoot({
      connectionString:
        'DefaultEndpointsProtocol=**********************************',
    }),
    CatModule,
  ],
  controllers: [AppController],
  providers: [AppService],
})
export class AppModule {}
```
**cat.module.ts:**
```
@Module({
  imports: [
    AzureTableStorageModule.forFeature(Cat, {
      table: 'Cat',
      createTableIfNotExists: true,
    }),
  ],
  providers: [CatService],
  controllers: [CatController],
})
export class CatModule {}
```
I got an error about a provider not available:
```
[Nest] 26448   - 12/18/2019, 8:41:44 AM   [AzureTableStorage] Create new TableService instance
[Nest] 26448   - 12/18/2019, 8:41:44 AM   [ExceptionHandler] Nest can't resolve dependencies of the AzureTableStorageRepository (AzureTableStorageService, ?). Please make sure that the argument AZURE_TABLE_STORAGE_NAME at index [1] is available in the AzureTableStorageModule context.

Potential solutions:
- If AZURE_TABLE_STORAGE_NAME is a provider, is it part of the current AzureTableStorageModule?
- If AZURE_TABLE_STORAGE_NAME is exported from a separate @Module, is that module imported within AzureTableStorageModule?
  @Module({
    imports: [ /* the Module containing AZURE_TABLE_STORAGE_NAME */ ]
  })
 +2ms
Error: Nest can't resolve dependencies of the AzureTableStorageRepository (AzureTableStorageService, ?). Please make sure that the argument AZURE_TABLE_STORAGE_NAME at index [1] is available in the AzureTableStorageModule context.

Potential solutions:
- If AZURE_TABLE_STORAGE_NAME is a provider, is it part of the current AzureTableStorageModule?
- If AZURE_TABLE_STORAGE_NAME is exported from a separate @Module, is that module imported within AzureTableStorageModule?
  @Module({
    imports: [ /* the Module containing AZURE_TABLE_STORAGE_NAME */ ]
  })

    at Injector.lookupComponentInParentModules (/home/kiwi/dev/azure-db/node_modules/@nestjs/core/injector/injector.js:190:19)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async Injector.resolveComponentInstance (/home/kiwi/dev/azure-db/node_modules/@nestjs/core/injector/injector.js:146:33)
    at async resolveParam (/home/kiwi/dev/azure-db/node_modules/@nestjs/core/injector/injector.js:100:38)
    at async Promise.all (index 1)
    at async Injector.resolveConstructorParams (/home/kiwi/dev/azure-db/node_modules/@nestjs/core/injector/injector.js:115:27)
    at async Injector.loadInstance (/home/kiwi/dev/azure-db/node_modules/@nestjs/core/injector/injector.js:79:9)
    at async Injector.loadProvider (/home/kiwi/dev/azure-db/node_modules/@nestjs/core/injector/injector.js:36:9)
    at async Promise.all (index 4)
    at async InstanceLoader.createInstancesOfProviders (/home/kiwi/dev/azure-db/node_modules/@nestjs/core/injector/instance-loader.js:41:9)
```

## Expected behavior
I expect to not have any error when I am using forRoot instead of .env

## Minimal reproduction of the problem with instructions
https://github.com/jogelin/nestjs-azure-database-sample/tree/issue-for-root

**Comment by manekinekko** [comment]
Hi @jogelin  this was fixed in a6b22dafcef4bdaff546a840b9bf183cb5cc6a0. Can you try again and lmk?

cheers.

**Comment by flexwie** [comment]
I have the same issue in v2.3.0. The problem seems to be that the service never gets passed the configuration when creating a new client for the table storage (see [here](https://github.com/nestjs/azure-database/blob/8c34af4472303d275932a781d399266a5de4a230/lib/table-storage/azure-table.service.ts#L17)). The storage service clients gets no connection string and therefore always falls back to the environment variables. a6b22da does not address this issue.
